src/CookieStore.ts

```diff
@@ -1,28 +1,26 @@
 import { Cookie, parse as parseCookie } from 'set-cookie-parser'
 
 interface RequestLike {
-    credentials: Request['credentials'];
-    url: string;
+  credentials: Request['credentials']
+  url: string
 }
 interface HeadersLike {
-    get(name: string): string | null;
+  get(name: string): string | null
 }
 interface ResponseLike {
-    headers: HeadersLike;
+  headers: HeadersLike
 }
 type Store = Map<string, StoreEntry>
 type StoreEntry = Map<string, Cookie>
-type CookieString = Omit<Cookie, 'expires'> & { expires?: string };
+type CookieString = Omit<Cookie, 'expires'> & { expires?: string }
 
 export const PERSISTENCY_KEY = 'MSW_COOKIE_STORE'
 
```

Comment:
I'd define the global variable here, determining if we can use local storage:

```js
const SUPPORTS_LOCAL_STORAGE = typeof localStorage !== 'undefined'
```

Reply:
If the localStorage is removed during runtime, it will fail. Using a getter avoid this case

Reply:
What is the case when the `localStorage` is removed during runtime? I'd say that's an edge-case scenario that indicates an error in the runtime, but I may be wrong.

Reply:
I don't have any idea of what can be the case.

But this is technically possible.

By the way, if we store the check in a const as you recommend, I'll need extra time to find a way to test undefined localStorage. Because it cannot work as it is currently done